dynacred/spec/calypso.spec.ts

```diff
@@ -1,8 +1,8 @@
-import DarcInstance from "@dedis/cothority/byzcoin/contracts/darc-instance";
-import { CalypsoReadInstance, CalypsoWriteInstance, OnChainSecretRPC, Write } from "@dedis/cothority/calypso";
-import { Darc, Rule } from "@dedis/cothority/darc";
-import Log from "@dedis/cothority/log";
-import { curve } from "@dedis/kyber";
+import DarcInstance from "@c4dt/cothority/byzcoin/contracts/darc-instance";
```

Comment:
So, what's the default now: do we want dynacred to deps on c4dt's {cothority,kyber} or dedis'?

Reply:
as soon as I get my stuff into DEDIS we'll do `make cothority_dedis`...